Mon2Cam.sh

```diff
@@ -76,6 +106,7 @@ then
 	exit 1
 fi
 
+
```

Comment:
Extra space

Reply:
Also fixed in 88921f42685b5046224bbe6771edd377275d6c27.